### ISSUE TITLE
Tech Discovery - How to release Toxic Exposure and new pdf version with a sticky feature flag

### ISSUE THREAD

**Issue by RakshindaAslam** [department-of-veterans-affairs/va.gov-team#79275]
## Issue Description
Toxic Exposure and the new pdf version need to be released together and to smaller subset of Veterans. This ticket is to understand how a user is consistently sent down the path of the Toxic Exposure flow across multiple sessions till submission via the TE feature flag.  

---
## Tasks
- [ ] Understand how flipper works - does it work for user accounts
- [ ] If not, how can we make TE sticky for all facets of user journey from FrontEnd to Backup submission
- [ ] Check with the Burial team

## Acceptance Criteria
- [ ] Does the flipper work with sticky path as currently implemented?
- [ ] If not, document path for implementing sticky path

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by aurora-a-k-a-lightning** [comment]
TL;DR: The Flipper feature flags are "sticky", but there may be some complications in the business logic layer and in the middle-tier "submit job" land that needs answering before we implement the new version of form 21-526EZ.

All is explained in this diagram:
<img width="1001" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL0JDQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--6d1903d251bcda53ef52d6c44282e2c3133a368e/DBex%20Diagrams%20-%20%2379275.jpeg" alt="DBex Diagrams - #79275.jpeg" />

Notes:
1. The Flipper feature flag _**does**_ stick with the user until the end i.e. the flag is, indeed, "sticky".
2. We are assuming the Veteran needs to know which form we are filing for them on their behalf.
3. How do we relay this information to Veterans?
4. Veterans who are chosen with the Flipper flag - will they have to start over the form? Or will a simple pop-up suffice?
5. Veterans who are chosen with the Flipper flag/if we decide to roll back and are in "submit land" - will we have to send them a message/email telling them to start over? Or skip over that Veteran based on knowing which form they filled out, which we have no way of doing yet.
6. We need a way to capture which version of the form the Veteran was filling out.

We are closing this ticket for now until another team that is in the midst of this process rolls out to get more answers to these questions.
